### DISCUSSION TITLE
Error Auth with Facebook Credentials

### DISCUSSION THREAD

**Discussion by Syscomp-Software** [firebase/flutterfire#9725] [Q&A]
I'm trying to login with facebook credentials, and I'm getting the return:

 [firebase_auth/invalid-credential] The supplied auth credential is malformed or has expired. [ Unable to retrieve user's identity from facebook.com response, JSON response does not contain id: {"picture":{"data":{"is_silhouette":false,"width":100,"url":"https://platform-lookaside.fbsbx.com/platform/profilepic/?asid=5300155540111150&height=100&width=100&ext=1668284612&hash=AeRnXj5Vc2DfvPh1yV4","height":100}}} ]

Does anyone help me?

**Comment by JoseMiguelCh** [comment]
Hi!, could you solve this? Im facing the same error
